core/vpgl/file_formats/vpgl_nitf_rational_camera.cxx

```diff
@@ -145,7 +145,6 @@ static int geostr_to_double(const char* in_string, double* val, vpgl_nitf_ration
     vcl_strncpy(temp,in_string-length,length);
     if ( (fsec = (float)vcl_atof(temp)) >= 60.0f || fsec<0.0f)
       return 0;
-    delete temp;
```

Comment:
Was getting rid of this line intentional?  It seems that a variable named temp is allocated three times in this file, but deleted only twice.